tests/rptest/e2e_tests/flink_scale_test.py

```diff
@@ -315,18 +322,28 @@ def test_transactions_scale_swarm(self):
         for flink in flinks:
             flink.start()
 
-        # Prepare topics
-        # Total number of workloads would be 2 x 5 = 10
-        workloads_per_node = 4
-        # For 5 nodes, total topics is 50 * 4 * 5 = 1000
-        # for 2xlarge 80 is a maximum
-        # 100 results in
-        # org.apache.flink.streaming.runtime.tasks.mailbox.TaskMailbox$MailboxClosedException: Mailbox is in state CLOSED, but is required to be in state OPEN for put operations.
-        topics_per_workload = 25
-        # 1 mil events total for single node, per topic
-        # Total number of events will be
-        # (1M / workloads) * topics_per_workload = 250k * 50 = 10M
-        target_total_events = 1 * 1024 * 1024
+        if not self.redpanda.dedicated_nodes:
+            # Prepare topics for docker env
+            # Total number of workloads would be 25
+            workloads_per_node = 1
+            topics_per_workload = 25
+            # 500k events total for single node, per topic
+            # Total number of events will be
+            # (500k / workloads) * topics_per_workload = 500k * 25 = ~12M
+            target_total_events = 500 * 1024
+        else:
+            # Prepare topics for EC2
+            # Total number of workloads would be 2 x 5 = 10
```

Comment:
Is this comment accurate?

total flink nodes = 5
workloads_per_node = 4

total = 20?

Reply:
Yes, this should be updated. Thanks

Reply:
total workloads 4 * 5 = 20
total topics 4 * 5 * 25 = 500